{
  "cve_id": "CVE-2024-28179",
  "timestamp": "2025-01-14 16:58:32",
  "phases": {
    "primary": {
      "completed": true,
      "timestamp": "2025-01-10 10:40:03"
    },
    "secondary": {
      "completed": true,
      "timestamp": "2025-01-11 20:55:17"
    },
    "extraction": {
      "completed": true,
      "timestamp": "2025-01-11 22:12:48"
    }
  },
  "urls": [
    {
      "url": "https://github.com/jupyterhub/jupyter-server-proxy/blob/9b624c4d9507176334b46a85d94a4aa3bcd29bed/jupyter_server_proxy/handlers.py#L433",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://github.com/jupyterhub/jupyter-server-proxy/commit/764e499f61a87641916a7a427d4c4b1ac3f321a9",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://github.com/jupyterhub/jupyter-server-proxy/commit/bead903b7c0354b6efd8b4cde94b89afab653e03",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://github.com/jupyterhub/jupyter-server-proxy/security/advisories/GHSA-w3vc-fx9p-wp4v",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    }
  ]
}